### ISSUE TITLE
Suggestion: Tooltips for Filter Mode Icons

### ISSUE THREAD

**Issue by hk21702** [RimSort/RimSort#483]
Add tooltips for filter mode icons.

![image](https://github.com/user-attachments/assets/28e9e5f7-ec66-4f8f-9cb7-19ca6f887b6f)

1) RimSort icon(black) = All Mods
2) Orange = Official Content (Core and DLC)
3) Dark blue Folder Icon = Local Mods (Including SteamCMD and Git)
4) Git Icon = Git Mods
5) Cloud with Steam icon = SteamCMD Mods
6) Steam Icon = Steam Mods

**Comment by oceancabbage** [comment]
Addressed in https://github.com/RimSort/RimSort/pull/494
